core/schema/src/main/java/org/hawkular/metrics/schema/SchemaService.java

```diff
@@ -50,56 +51,91 @@ public void run(Session session, String keyspace, boolean resetDB, int replicati
     public void run(Session session, String keyspace, boolean resetDB, int replicationFactor, boolean updateVersion) {
         CassalogBuilder builder = new CassalogBuilder();
         Cassalog cassalog = builder.withKeyspace(keyspace).withSession(session).build();
-        Map<String, ?> vars  = ImmutableMap.of(
+        URI script = getScript();
+        Map<String, ?> vars = this.getVars(session, keyspace, resetDB, replicationFactor);
+        cassalog.execute(script, this.getTags(), vars);
```

Comment:
We want to update the schema version after each update is applied. If ```updateVersion``` is true then you will want to use the new version of the execute method, 
```java
void execute(URI script, Collection tags, Map vars, ChangeSetListener listener)
```

Reply:
I went back and looked at my notes in Trello to see why I thought we needed to use the ChangeSetListener. It is not clear to me now, but more importantly, I agree with your assessment that it is not needed.